tests/libcxx/expected_results.txt

```diff
@@ -522,49 +522,49 @@ std/language.support/support.limits/limits/numeric.limits.members/traps.pass.cpp
 
 
 # *** STL BUGS ***
-# STL bug: VSO-121977 "<locale>: the enum value of std::money_base is not correct[libcxx]"
+# STL bug: GH-1112 VSO-121977 "<locale>: the enum value of std::money_base is not correct[libcxx]"

```

Comment:
"STL bug: " seems redundant in a section entitled "STL BUGS". Do we really need a reminder every few lines?